File: Swift/L3/Lista_03_EX04/Usuario.swift
import Foundation
 
 public class Usuario{
 
    public var login: Login
     
    init(login: Login) {
        self.login = login
    }
  
	get pegaLogin()
	{
		return login
	}
	
	func boasVindas( ) {
		print("Bem-vindo: \ (login)")
	}
	
}
	

File: Java/Lista 4/L4_Ex1.1/src/l4_ex1/pkg1/Usuario.java

package l4_ex1.pkg1;

import java.util.Scanner;

public class Usuario {
    private String login;

    public Usuario(String login) {
        this.login = login;
    }

    public String getLogin() {
        return login;
    }
    
    public int menu(){
        int opcao = 0;
        Scanner ler = new Scanner( System.in );
        System.out.println("Menu");
        return opcao;
    }
    
    public void boasvindas(){
        System.out.println("Bem-vindo " + this.login );
    }
}
    
